galaxy.yml

```diff
@@ -25,6 +25,12 @@ documentation: "https://github.com/prometheus-community/ansible/blob/main/docs"
 homepage: "https://prometheus.io"
 issues: "https://github.com/prometheus-community/ansible/issues"
 build_ignore:
-  - 'tests/*'
+  - .config
+  - .github
+  - .gitignore
+  - changelogs
+  - docs
```

Comment:
```suggestion
```
I don't think we should remove docs from the build, since we can publish docs with the collection that can then be read with the `ansible-doc` command after install.